app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -97,12 +99,13 @@ public function store(Request $request, $id)
         }
 
         $sig_filename = '';
+
         if ($request->filled('signature_output')) {
             $sig_filename = 'siglog-'.Str::uuid().'-'.date('Y-m-d-his').'.png';
             $data_uri = e($request->input('signature_output'));
             $encoded_image = explode(',', $data_uri);
             $decoded_image = base64_decode($encoded_image[1]);
-            Storage::put('private_uploads/signatures/'.$sig_filename, (string) $decoded_image);
+            $path = Storage::put('private_uploads/signatures/'.$sig_filename, (string) $decoded_image);
```

Comment:
I don't think we're even using this $path variable tbh - I could probably take that out